utils/airtable_es2015.js

```diff
@@ -107,12 +107,6 @@ var hydrateFromAirtable = (exports.hydrateFromAirtable = async function hydrateF
     dataStore.multipleChoiceOptions,
     "variable_name"
   );
-  replaceId(
```

Comment:
This is unrelated, but I removed it because we aren't using this column any more